games-vue-client/src/components/chat/chatStore.js

```diff
@@ -0,0 +1,122 @@
+import Socket from "@/socket";
+
+const chatStore = {
+  namespaced: true,
+  state: {
+    chats: {
+      'server': {
+        chatId: 'server',
+        users: [
+          { "playerId": "431", "name": "RandomCat79" },
+          { "playerId": "123", "name": "Zomis" }
+        ],
+        messages: [
+          { message: 'Hello', from: "RandomCat79", date: Date.now() },
+          { message: 'Hello there!', from: "Zomis", date: Date.now() },
+        ],
+      },
+      'game-UR': {
+        chatId: 'game-UR',
+        users: [
+          { "playerId": "431", "name": "RandomCat79" },
+          { "playerId": "123", "name": "Zomis" },
+          { "playerId": "444", "name": "Urmakaren" },
+        ],
+        messages: [
+          { message: 'Hello UR', from: "RandomCat79", date: Date.now() },
+          { message: 'Hello you!', from: "Urmakaren", date: Date.now() },
+        ],
+      },
+      'game-Dixit': {
+        chatId: 'game-Dixit',
+        users: [
+          { "playerId": "123456", "name": "JKR" },
+          { "playerId": "123", "name": "Zomis" },
+        ],
+        messages: [
+          { message: 'Hello Zomis', from: "JKR", date: Date.now() },
+          { message: 'Hello you!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you2!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you3!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you4!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you5!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you6!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you7!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you8!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you9!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you10!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you11!', from: "Zomis", date: Date.now() },
+          { message: 'Hello you12!', from: "Zomis", date: Date.now() },
+        ],
+      }
+    }
+  },
+  getters: {
+    chats: state => { return state.chats }
+  },
+  mutations: {
+    addChat(state, data) {
+      const { chatId, users } = data;
+      state.chats[chatId] = { chatId, users }
```

Comment:
This needs to use `Vue.set` (see `DslGameState.js` for an example) in order to make the property reactive.

Reply:
Oh, I didn't see that one yet, nice!